include/aikido/planner/vectorfield/VectorFieldPlannerExceptions.hpp

```diff
@@ -9,18 +9,33 @@ namespace aikido {
 namespace planner {
 namespace vectorfield {
 
-/// Define runtime error for the termination of vectorfield planner.
-class VectorFieldTerminated : public std::runtime_error
+/// Define termination of vectorfield planner.
+class VectorFieldTerminated : public std::exception
```

Comment:
`VectorFIeldTerminated` defines the end of the planning, which implies mission completed. I throw it as an exception, so that the integrator could be stopped. The input integral time is usually larger than it should be. 

`VectorFieldError` defines the error found in the planning process, so it inherits `std::runtime_error`.